### DISCUSSION TITLE
are all LENA metrics listed on readthedocs?

### DISCUSSION THREAD

**Discussion by alecristia** [LAAC-LSCP/ChildProject#408] [Q&A] [answered]
[Readthedocs](https://childproject.readthedocs.io/en/latest/metrics.html#lena-metrics) lists 16 LENA metrics, without including AWC (Fem + Mal AWC) or CTC. Is that right?

**Comment by LoannPeurey** [comment, marked as answer]
Thank you, I must have forgotten to add them. Fixed now =) [lena metrics](https://childproject.readthedocs.io/en/latest/metrics.html#lena-metrics).

**Reply by alecristia** [reply]
@LoannPeurey should we also mention duration_its?

**Reply by LoannPeurey** [reply]
Good question, all of the duration_set fields are always present when extracting metrics but are not listed in the documentation as metrics as their function is quite different from other metrics that are all related to actual annotation value. They are given mainly to have a way of changing between 'per hour' values and absolute values. It is also handy to know what quantity of audio was annotated for each metric line.

I think the only documented info on those columns is in this sentence : "The duration_<set> columns contain the total amount of annotated time covering each time-bin and each set, in milliseconds." . Perhaps we should include an explanation in each of the metric types (lena aclew custom ...)

**Reply by alecristia** [reply]
perhaps by adding in each section a reminder "these are in addition to the metrics that are always extracted" with a link to the explanations on those, in a new section. Right now, the info about these columns is in the middle of the [overview](https://childproject.readthedocs.io/en/latest/metrics.html#overview) section, which I fear people will miss. I also think we should add "e.g., duration_its, duration_vtc" etc with all the duration metrics of this type we have now, since some people will do ctrl+F for the name ;)
